cairo/tests/utils/strategies.py

```diff
@@ -446,7 +468,10 @@ def bounded_u256_strategy(min_value: int = 0, max_value: int = 2**256 - 1):
             )
         ),
         _snapshots=st.builds(list, st.just([])),
-        created_accounts=st.sets(address, max_size=10),
+        created_accounts=st.sets(
+            address.filter(lambda x: x not in [SYSTEM_ADDRESS, BEACON_ROOTS_ADDRESS]),
```

Comment:
isn't address already not in [SYSTEM_ADDRESS, BEACON_ROOTS_ADDRESS] from line 442?